sphincsplus_cgo/binding.go

```diff
@@ -1,9 +1,9 @@
 package sphincsplus
 
-// #cgo CFLAGS: -g -I/usr/local/include/sphincsplus
-// #cgo LDFLAGS: -L/usr/local/lib/sphincsplus/ -Wl,-rpath,/usr/local/lib/sphincsplus/ -lsphincsplus
+// #cgo CFLAGS: -g -I../sphincsplus/ref/ -D PARAMS=sphincs-shake-256f
+// #cgo LDFLAGS: -L ../sphincsplus/ref/ -l:libsphincsplus.a
 // #define PARAMS sphincs-shake-256f
```

Comment:
I think with `-D PARAMS=` above you can remove this `// #define PARAMS`, but I'm not sure.